lib/logstash/filters/multiline.rb

```diff
@@ -17,15 +17,15 @@
 # [source,ruby]
 #     filter {
 #       multiline {
-#         type => "type"
 #         pattern => "pattern, a regexp"
 #         negate => boolean
 #         what => "previous" or "next"
 #       }
 #     }
 #
-# The `pattern` should be a regexp which matches what you believe to be an indicator
-# that the field is part of an event consisting of multiple lines of log data.
+# The `pattern` should be a regexp (<<plugins-filters-grok,grok>> patterns are
```

Comment:
@magnusbaeck this works. This is asciidoc macros. `plugins-filters-grok` is the ID which is defined  in that page.